www/rules/index.html

```diff
@@ -105,7 +105,20 @@ <h4>7. Naming your map</h4>
     </li>
   </ul>
 
-  <h2>🗃️ Server Type Rules</h2>
+  <h2>🗃️ Hosting & Server Type Rules</h2>
+
+  <h4>🌐 Hosting rules</h4>
+  <ul>
+    <p>Hosting publicly accessible servers requires them to be registered with Teeworlds' and DDNet master servers.
+      Game servers that register with these master servers are subject to <a href="https://www.teeworlds.com/?page=docs&wiki=rules/server_rules">the server hosting rules of Teeworlds</a> as well as the additional DDNet rules:</p>
```

Comment:
This reads wrong. The "DDNet rules" below are currently copied from the link, those are not additional rules except the last one.